test/e2e/case20_delete_objects_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"open-cluster-management.io/config-policy-controller/test/utils"
+)
+
+const (
+	case20ConfigPolicyNameCreate string = "policy-pod-create"
+	case20ConfigPolicyNameEdit   string = "policy-pod-edit"
+	case20PolicyYamlCreate       string = "../resources/case20_delete_objects/case20_create_pod.yaml"
+	case20PolicyYamlEdit         string = "../resources/case20_delete_objects/case20_edit_pod.yaml"
+)
+
+var _ = Describe("Test status fields being set for object deletion", func() {
```

Comment:
Good start to tests here - I think I will want another test to make sure that editing an object will not populate `createdByPolicy` (the current policy does not make any changes to the object). Based on the code, I believe this will pass, it's just a matter of completeness here.

This test can be added in another PR.